DOCUMENTATION.md

```diff
@@ -0,0 +1,113 @@
+# Documentation
+
+This is the documentation of the SDALib library, the layout of this document is the following:
+- SDAData
+  - Car
+  - Situation
+  - Tickcount
+- SDAAction
+  - Steer
+  - Accel
+  - Brake
+  - Gear 
+- SDADriver
+- SDASimulate
+- Example
+
+## SDAData
+
+A class that contains all of the data needed for a driver to understand the gamestate. It contains a Car struct, a Situation struct and a tickcount.
+
+### Car
+
+This struct stores all data from the car. Also stores the segments that the car is on and where it is going.
+
+### Situation
+
+This struct stores all data from the current situation.
```

Comment:
Do we have (somewhere) documentation on what is actually contained in these structs, besides the fact the struct contain 'the data', such as: what variables are available, what are some common variables that one might want to use (speed, position relative to road), and which variables aren't available (because they're pointers)

Reply:
not yet, should be added in the future to a documentation file and I will link that file when it is made.